doc/nest-03.md

```diff
@@ -0,0 +1,35 @@
+# Installing and uninstalling a package
+
+## Nest pull
+
+`nest pull` synchronizes the local packages' cache with the one on Raven's server. You need to **keep this local cache up to date as often as possible**, or you won't be able to install the packages or versions that came out after your last cache update. This is very important. Do not forget that.
+
+## Nest install
+
+To install packages, run `nest install <package>`. You can even put a requirement instead of the package's name. Nest does a lot of things when installing a package, let's go over each one of them.
+
+[//]: # (TODO: add link to the list of packages of Nest)
+[//]: # (TODO: add link to the first chapter)
+First, Nest checks if the package exists in the local cache. If it does, great ! Nest continues its work. If it doesn't, you might need to run `nest pull` to refresh your cache first, or maybe the package you requested doesn't exist at all. You might want to check your syntax, and you can see a list of all available packages for Nest [here]() in the official repositories. Nest determines after that if there are any dependencies to install by going through the dependency graph, as explained in the [first chapter](), and will install them as well.
```

Comment:
You can suggest to use `nest search` to look for packages through the command line, or the website (in progress) to look for packages using a gui.